cmd/thanos/query_frontend.go

```diff
@@ -43,7 +43,9 @@ type queryFrontendConfig struct {
 
 func registerQueryFrontend(app *extkingpin.App) {
 	comp := component.QueryFrontend
-	cmd := app.Command(comp.String(), "query frontend")
+	cmd := app.Command(comp.String(), "query frontend command implements a service that can be put in front of Thanos Queriers"+
+		" to improve the read path. It is based on the Cortex Query Frontend component so you can find"+
+		" some common features like Splitting and Results Caching. Query Frontend is fully stateless and horizontally scalable.")
```

Comment:
The description should be more concise. What about `a service deployed in front of queriers to improve query parallelisation and caching` ?

Reply:
ok thx! I update it now